tests/test_redis_wq.py

```diff
@@ -0,0 +1,109 @@
+import unittest
+import sys
+from unittest.mock import patch
+from mediaire_toolbox.queue.redis_wq import RedisWQ
+
+
+class MockRedis():
+    def __init__(self):
+        self.hashmap = {}
+        self.expiremap = {}
+
+    def incr(self, key):
+        if key in self.hashmap:
+            new_value = int.from_bytes(self.hashmap[key], sys.byteorder) + 1
+            self.hashmap[key] = new_value.to_bytes(1, sys.byteorder)
+        else:
+            self.hashmap[key] = (1).to_bytes(1, sys.byteorder)
+        return self.hashmap[key]
+
+    def get(self, key):
+        if key in self.hashmap:
+            return self.hashmap[key]
+        else:
+            return None
+
+    def expire(self, key, time):
+        self.expiremap[key] = time
+
+    def execute(self):
+        pass
+
+    def pipeline(self):
+        return self
+
+    def rpoplpush(self, src, dst):
+        value = self.hashmap[src].pop()
+        self.hashmap[dst].append(value)
+        return value
+
+    def setex(self, *args):
+        pass
+
+
+class TestRedisWQ(unittest.TestCase):
+    def setUp(self):
+        self.mock_redis = MockRedis()
+        self.r_wq = RedisWQ(name='mock_limit_rate', db=self.mock_redis)
+
+    def test_get_expiry_time_sec(self):
+        self.assertEqual(RedisWQ._get_expirytime('sec'), 1)
+
+    def test_get_expiry_time_min(self):
+        self.assertEqual(RedisWQ._get_expirytime('min'), 60)
+
+    def test_get_expiry_time_hour(self):
+        self.assertEqual(RedisWQ._get_expirytime('hour'), 60*60)
+
+    def test_get_expiry_time_invalid(self):
+        self.assertEqual(RedisWQ._get_expirytime('invalid'), 60)
```

Comment:
is that the expected result when i pass an invalid unit? shouldnt we raise an error?

Reply:
I was thinking about setting it to default when passing an invalid unit, but on second thoughts, raising an error would indeed cause less confusion